static/app/components/teamSelector.tsx

```diff
@@ -95,12 +95,15 @@ const placeholderSelectStyles: StylesConfig = {
 
 type Props = {
   onChange: (value: any) => any;
-  organization: Organization;
   /**
    * Controls whether the dropdown allows to create a new team
    */
   allowCreate?: boolean;
   includeUnassigned?: boolean;
+  /**
+   * TODO: It can be undefined in some cases -> needs investigation
```

Comment:
```suggestion
   * TODO(arthur): It can be undefined in some cases -> needs investigation
```
We usually put author's name in the parenthesis so that:
1. it's easier to search for your TODOs
2. we create some accountability